lib/persisted-model.js

```diff
@@ -641,7 +641,8 @@ module.exports = function(registry) {
       accessType: 'WRITE',
       accepts: {arg: 'id', type: 'any', description: 'Model id', required: true,
                 http: {source: 'path'}},
-      http: {verb: 'del', path: '/:id'}
+      http: {verb: 'del', path: '/:id'},
+      returns: {arg: 'count', type: 'number', root: true}
```

Comment:
>Then the return type is an object instead of number.

I see, I thought it was describing the `count` property. Changing to `object`.

>What about updateAll? It should have similar return value.

Beyond the scope of my current task, but IMO yes and we should probably check the others too. Like the same can go for destroyAll (count of items deleted), updateAll, bulkUpdate, maybe create a new issue to find out what needs updating?